superset/assets/src/CRUD/CollectionTable.jsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
```

Comment:
I think you could simplify this component significantly by separating the table component from the row component (ie different files instead of `this.renderHeaderRow()`, etc.)

Reply:
The file is ~200 lines which is still pretty intelligible to me. Though I agree we should break down the larger files. I know I tend to make large modules :'(